specification/sql/resource-manager/Microsoft.Sql/preview/2022-08-01-preview/examples/RefreshExternalGovernanceStatus.json

```diff
@@ -0,0 +1,29 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "sqlcrudtest-7398",
+    "serverName": "sqlcrudtest-4645",
+    "api-version": "2022-08-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "properties": {
+          "requestId": "00000000-0000-0000-0000-000000000000",
+          "requestType": "UpdatePurviewMetadata",
+          "queuedTime": "2/12/2022 8:33:27 PM",
+          "serverName": "testsvr.database.windows.net",
+          "status": "Completed"
+        },
+        "id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/myResourceGroup/providers/Microsoft.Sql/locations/japaneast/refreshExternalGovernanceStatusOperationResults/9d9a794a-5cec-4f23-af70-d29511b522a4",
+        "name": "9d9a794a-5cec-4f23-af70-d29511b522a4",
+        "type": "Microsoft.Sql/servers/refreshExternalGovernanceStatusOperationResults"
```

Comment:
You should not be reutrning full objects as the response for a POST call , it is better to respond with some details on the status of the operation

Reply:
Which details do you think I should omit? Currently, the response does contain the status of the operation.

The return type is similar to:
https://learn.microsoft.com/en-us/rest/api/sql/2021-02-01-preview/servers/import-database?tabs=HTTP

Reply:
ok that makes sense. 
Although you need to fix the type here, the id indicates that this is a resource of type : 
"Microsoft.Sql/locations/refreshExternalGovernanceStatusOperationResults"

 but you have defined it as         "type": "Microsoft.Sql/servers/refreshExternalGovernanceStatusOperationResults"

Please fix either the Id or type as appropriate. The example you have shared has deined it correctly in the sample response.

Reply:
Thanks, fixed.